pages/ai-models/together-ai.mdx

```diff
@@ -0,0 +1,158 @@
+---
+title: Together AI Integration
+description:
+  The Together AI Integration in BuildShip enables users to connect their Together AI accounts and streamline AI-powered
+  workflows with ease.
+---
+
+import Image from 'next/image';
+import togetheraiOg from '/public/integrations/ai-models/togetheraiOg.png';
+import togetherai1 from '/public/integrations/ai-models/togetherai1.png';
+import togetherai2 from '/public/integrations/ai-models/togetherai2.png';
+import togetherai3 from '/public/integrations/ai-models/togetherai3.png';
+import togetherai4 from '/public/integrations/ai-models/togetherai4.png';
+import { Callout } from 'nextra/components';
+
+# Together AI Integration
+
+The Together AI integration in BuildShip allows users to seamlessly connect with Together AI’s API to leverage powerful
+generative AI capabilities. Users can easily generate embeddings, text completions, and chat-based completions,
+incorporating advanced AI models into their workflows with minimal effort. Together AI provides access to 100+
+open-source models through its serverless Inference Engine, enabling efficient use of leading models for tasks such as
+text generation and embedding creation directly from BuildShip.
+
+<br />
+
+<Image src={togetheraiOg} alt='Together AI Integration' width={1200} />
+
+By utilizing BuildShip's Together AI nodes, users can simplify the integration of AI-powered features into their
+applications. This documentation will guide you through configuring API keys and working with the provided nodes to
+fully integrate these AI services.
+
+## Prerequisites ✅
+
+Before you can start using the Together AI Nodes in BuildShip, ensure you have the following prerequisites in place:
+
+### Together AI Account
+
+You will need an active Together AI account to utilize this integration. If you do not have an account, you can create
+one at [Together AI](https://together.ai/).
+
+### Together AI API Authentication
+
+You will need a Together AI API Key to authenticate your requests. For more information on obtaining an API Key, refer
+to the [Together AI API documentation](https://docs.together.ai/authentication).
+
+<br />
+
+<Image src={togetherai1} alt='Together AI Integration' width={1200} />
+
+- **API Key**: The Together AI API key is required to authenticate each request you make through the BuildShip nodes.
```

Comment:
> You will need a Together AI API Key to authenticate your requests. For more information on obtaining an API Key, refer to the [Together AI API documentation](https://docs.together.ai/authentication).

> - **API Key**: The Together AI API key is required to authenticate each request you make through the BuildShip nodes.

Perhaps we could combine these two and remove the single list item entirely?

> ## Set API Key 
>Before using any of the Together AI nodes, set your API key for authentication.
>
> ### Inputs
>
> - **API Key**: The API key obtained from Together AI for authentication. Make sure to keep it secure.
>
> <Callout>
>   You must provide the API key in the headers of each node request. Once set, it allows secure communication with the Together AI API.
> </Callout>

Lets remove the "Set API Key" section, instead refer the prerequisites section for the API Key if needed. We can also mention that the users can save their API Keys as Secrets and add a link to [/secrets](https://docs.buildship.com/secrets).